app/styles/global/_base.scss

```diff
@@ -1,3 +1,8 @@
 // - - - - - - - - - - - - - - - - - - -
 // - - base
 // project defaults for base elements - h1-h6, p, a, etc.
+
+html, body {
+  height:100%;
```

Comment:
is the height set to 100% for the desktop window.  What are you trying to achieve?  `height: 100vh` might be a better solution if you are trying to fill the screen.